src/index.html

```diff
@@ -1,13 +1,48 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <title>Product cards</title>
+    <link rel="stylesheet" href="./style.css">
+  </head>
+  <body>
+    <div class="card-container" data-qa="card">
+      <img
+        src="images/imac.jpeg"
+        alt="iMac photo"
+        class="card-photo"
+      >
+      <div class="card-content">
+        <p class="product-name">
+          Моноблок APPLE A1419 iMac 27" Retina 5K (MNED2UA/A)
+        </p>
+        <p class="product-code">Код товара: 195434</p>
+      </div>
+      <div class="product-rating">
+        <div class="product-rating__stars">
+          <span class="star-active"></span>
```

Comment:
```suggestion
          <span class="stars__star"></span>
```
create block stars and 1 star should be its element